src/Symfony/Component/Serializer/Encoder/ChainDecoder.php

```diff
@@ -32,6 +32,8 @@ public function __construct(array $decoders = array())
 
     /**
      * {@inheritdoc}
+     * 
+     * @throws RuntimeException If no decoder is found.
```

Comment:
>@throws Exception doesn't help here anyway. If you handle specific exception, you know internals of Serializer.

I'm not following, to me there's a big difference between `\Exception` and `\Symfony\Component\Serializer\Exception\Exception`, because with the later I know from where the exception comes from and I may handle things differently knowing that.